---
layout: page
title: Resources
subtitle: Potentially helpful resources.
---

## Templates

* [Trello Template: Scientific Research Project](https://trello.com/templates/project-management/scientific-research-project-(template)-GBnhFbVr/blakewilson16/recommend)
* [Research Article Outline Template - MS Word](https://docs.google.com/document/d/1Lnm4vBguRAh71Nem0pjfa67WBuhIPRp1/edit?usp=sharing&ouid=101304454590803644776&rtpof=true&sd=true)
* [Review Article Outline Template - MS Word](https://docs.google.com/document/d/1fz8BFixCmFOKDLaDvGhAnLZhJt-CpYMc/edit?usp=sharing&ouid=101304454590803644776&rtpof=true&sd=true)

## Databases and Repositories

* Proteins (structure, function, interactions, etc.):
  * [RCSB Protein Data Bank](https://www.rcsb.org/) - protein structures (often a good starting point for atomistic molecular dynamics simulations)
  * [Uniprot](https://www.uniprot.org/) - protein sequence and function
  * [BCL2DB](https://bcl2db.lyon.inserm.fr/BCL2DB/) - sequence, structural, and functional data for Bcl-2 and BH3 containing proteins (involved for example in mitochondrial apoptosis regulation)
  * [STRING](https://string-db.org/) - Protein-protein interaction networks and functional enrichment analysis.
  * [BindingDB](https://www.bindingdb.org/bind/index.jsp) - binding affinities for proteins and drug-like molecules
  * [Human Protein Atlas](https://www.proteinatlas.org/) - data on the human proteome
* Small molecules (drugs and other biologically relevant molecules):
  * [ChEBI](https://www.ebi.ac.uk/chebi/init.do) - small chemicals of biological interest
  * [NeuroPep](http://isyslab.info/NeuroPep/) - information on neuropeptides
  * [DrugBank](https://go.drugbank.com/) - drugs and drug products
  * [Swiss Lipids](http://www.swisslipids.org/#/) - lipids and lipid biology
* Biological pathways:
  * [Reactome](https://reactome.org/) - biological pathways
  * [Pathway Commons](https://www.pathwaycommons.org/) - biological pathway and interaction data
  * [KEGG PATHWAY](https://www.genome.jp/kegg/pathway.html) - biological pathway maps
* Biological models:
  * [BioModels](https://www.ebi.ac.uk/biomodels/) - mathematical models of biological systems
* Biological quantities:  
  * [Bionumbers](https://bionumbers.hms.harvard.edu/search.aspx) - broad range of biological/molecular biology quantities
* Cancer:
  * [COSMIC](https://cancer.sanger.ac.uk/cosmic) - somatic mutations in cancer
* Genomics:
  * [KEGG](https://www.genome.jp/kegg/) - genes and genomes
  * [PharmGKB](https://www.pharmgkb.org/) - pharmacogenomics information
  * [GEO](https://www.ncbi.nlm.nih.gov/geo/) - functional genomics data
  * [OMIM](https://www.omim.org/) - human genes and genetic disorders
  * [ENCODE](https://www.encodeproject.org/) - functional genomics and characterization data

## Software
* molecular dynamics simulation, docking, molecular visualization, etc.:
  * [LAMMPS](https://www.lammps.org/) - classical molecular dynamics simulator
  * [GROMACS](https://www.gromacs.org/) - classical molecular dynamics simulator and analysis
  * [NAMD](https://www.ks.uiuc.edu/Research/namd/) - classical molecular dynamics simulator targeted to biomolecular systems
  * [VMD](https://www.ks.uiuc.edu/Research/vmd/) - molecular visualization and analysis
  * [CHARMM-GUI](https://www.charmm-gui.org/) - molecular dynamics simulation system construction and input generation
  * [CaFE](https://github.com/HuiLiuCode/CaFE_Plugin) - plugin for VMD providing an interface for MM/PBSA analysis to compute binding free energies
  * [APBS](https://www.poissonboltzmann.org/) - Adaptive Poisson-Boltzmann Solver for MM/PBSA calculations
  * [MDAnalysis](https://www.mdanalysis.org/) - Python library for molecular dynamics simulation analysis
  * [MDTraj](https://www.mdtraj.org/1.9.8.dev0/index.html) - Python library for molecular dynamics simulation analysis
  * [Avogadro](https://avogadro.cc/) - Visualization, molecule editing, and simulation
  * [PyMOL](https://pymol.org/2/) - Python library for molecular visualization
  * [ngliew](https://github.com/nglviewer/nglview) - Jupyter/IPython widget to render and view molecular structures inside notebooks.
  * [KFC Server](https://mitchell-web.ornl.gov/KFC_Server/index.php) - protein-protein binding hotspot prediction
  * [ClusPro](https://cluspro.bu.edu/login.php) - protein-protein docking
  * [SWISS-MODEL](https://swissmodel.expasy.org/) - protein homology modeling
  * [Rosetta](https://www.rosettacommons.org/software) - protein structure analysis, docking, homology modeling, and more.
* biological pathway/reaction network/systems biology modeling:
  * [PySB](https://pysb.org/) - rules-based reaction network model specification and simulation using the Python programming language (ODE or SDE models)
  * [BioNetGen](https://bionetgen.org/) - rules-based reaction network model specification and simulation (ODE or SDE models)
  * [SBML](https://synonym.caltech.edu/) - reaction network model specification
  * [Smoldyn](https://www.smoldyn.org/) - particle-based spatial stochastic simulation of reaction networks
  * [NEURON](https://www.neuron.yale.edu/neuron/) - reaction-diffusion models and simulation (geared towards computational models of neurons and can be used as a Python library)
  * [Catalyst.jl](https://juliapackages.com/p/catalyst) -  reaction network model specification and generation using the Julia programming language (ODE, SDE, or jump models)
